frontend/src/locales/en-CH-scout.json

```diff
@@ -8,5 +8,10 @@
   },
   "global": {
     "language": "English (Scout)"
+  },
+  "contentNode": {
+    "storycontext": {
+      "name": "Story concept"
```

Comment:
Würde ich jetzt nicht hier anders übersetzen als in en.json

Reply:
Ich habe das einfach überall angeglichen. So dass es in der Standardsprache "Geschichte" heisst und im "Pfadideutsch" "Roter Faden"